environments/staging/postgresql.yml

```diff
@@ -12,29 +8,41 @@ override:
 
 dbs:
   main:
+    pgbouncer_host: pgproxy0
```

Comment:
I think it could get confused with `plproxy` on envs that have it so maybe call it `pgbouncer0`?

Reply:
Ah, well on staging I'm just using the pgbouncer that's already _on_ the plproxy machine (`pgproxy0-staging`), so in this case that is what I'm doing. On production though that's a good suggestion; I'll make a separate pgbouncer0 so not all traffic's going through one machine.

I'll probably have the partition dbs continue to use pgbouncer on the plproxy machine, since anything else would require kind of useless bouncing around between machines.